proxy/proxy.go

```diff
@@ -281,11 +281,17 @@ func addCorsHeaders(header http.Header) {
 // handlePaymentRequired returns fresh challenge header fields and status code
 // to the client signaling that a payment is required to fulfil the request.
 func (p *Proxy) handlePaymentRequired(w http.ResponseWriter, r *http.Request,
-	serviceName string) {
+	serviceName string, servicePrice int64) {
 
 	addCorsHeaders(r.Header)
 
-	header, err := p.authenticator.FreshChallengeHeader(r, serviceName)
+	// Use the default price of 1 satoshi if servicePrice is zero
+	price := servicePrice
+	if servicePrice == 0 {
+		price = 1
```

Comment:
nit: would be nice to define this as a constant.

Reply:
Cool yeah, good idea 👍 Where would be a good place to define this constant?

Reply:
Seems weird to have the constant defined in proxy.go? for now i will define it in proxy/service.go. But let me know if I should move it 👍

Reply:
I think for now just at the top of the same file.

Reply:
Ok cool 👍